<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpMeDecision</class>
 <widget class="QWidget" name="HelpMeDecision">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HelpMeDecision</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>755</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_wait">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>361</width>
       <height>381</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>正在分析.......

(该分析结果由系统
自动判定，仅供参考)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_main">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>636</width>
          <height>1000</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1000</height>
         </size>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>参考指标：</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_reference">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>20</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>综合整体(默认)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>投诉率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>手续规范率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>发放贷款效率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>归集出错率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>贷款出错率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>归集资金到账率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>贷款资金到账率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>网络缴存出错率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>网络提取出错率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>网络贷款出错率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>网络还款出错率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>网络逾期出错率</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>网络负时效概率</string>
          </property>
         </item>
        </widget>
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>200</y>
           <width>631</width>
           <height>461</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <column>
          <property name="text">
           <string>机构编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>机构名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>评级结果</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>考核周期</string>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>311</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>金融机构绩效评级结果(已填写考核表)：</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_fresh">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>670</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>刷新数据</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>80</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>搜索：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_search">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>80</y>
           <width>241</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>搜索机构相关信息</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>720</y>
           <width>611</width>
           <height>141</height>
          </rect>
         </property>
         <property name="title">
          <string>修改评级结果（综合结果）</string>
         </property>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>机构编号：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_id">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>30</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>40</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>评级结果：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_levelAlter">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>30</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>D</string>
           </property>
          </item>
         </widget>
         <widget class="QPushButton" name="pushButton_alter">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>90</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="pushButton_commit">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>890</y>
           <width>211</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>采用此决策评级结果并公布</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
